ipatests/test_integration/test_adtrust_install.py

```diff
@@ -303,3 +306,588 @@ def test_ipa_user_s4u2self_pac(self):
+        password
+        """
+        password = "wrong_pwd"
+        msg = (
+            "Must have Kerberos credentials to setup AD trusts on server: "
+            "Major (458752): No credentials were supplied, or the credentials "
+            "were unavailable or inaccessible, Minor (2529639053): "
+            "No Kerberos credentials available (default cache: KCM:)\n"
+        )
+        self.master.run_command(["kdestroy", "-A"])
+        result = self.master.run_command(
+            ["ipa-adtrust-install", "-A", "admin", "-a",
+             password, "-U"], raiseonerr=False
+        )
+        assert msg in result.stderr_text
+        assert result.returncode != 0
+
+    def test_adtrust_install_with_invalid_rid_base_value(self):
+        """
+        Test to check adtrust install with invalid rid-base
+        value
+        """
+        rid_base_value = "103.2"
+        msg = (
+            "ipa-adtrust-install: error: option " "--rid-base: "
+            "invalid integer value: '{}'"
+        ).format(rid_base_value)
+        result = self.master.run_command(
+            [
+                "ipa-adtrust-install",
+                "-A",
+                "admin",
+                "-a",
+                self.master.config.admin_password,
+                "--rid-base",
+                rid_base_value,
+                "-U",
+            ],
+            raiseonerr=False,
+        )
+        assert msg in result.stderr_text
+        assert result.returncode != 0
+
+    def test_adtrust_install_with_invalid_secondary_rid_base(self):
+        """
+        Test to check adtrust install with invalid secondary rid-base
+        value
+        """
+        sec_rid_base_value = "103.2"
+        msg = (
+            "ipa-adtrust-install: error: option "
+            "--secondary-rid-base: invalid integer value: '{}'"
+        ).format(sec_rid_base_value)
+        result = self.master.run_command(
+            [
+                "ipa-adtrust-install",
+                "-A",
+                "admin",
+                "-a",
+                self.master.config.admin_password,
+                "--secondary-rid-base",
+                sec_rid_base_value,
+                "-U",
+            ],
+            raiseonerr=False,
+        )
+        assert msg in result.stderr_text
+        assert result.returncode != 0
+
+    def test_adtrust_reinstall_updates_ipaNTFlatName_attribute(self):
+        """
+        Test checks that reinstalling ipa-adtrust-install with
+        new netbios name reflects changes in ipaNTFlatName attribute
+        and ipa trustconfig-show also reflects the same.
+        """
+        netbios_name = "TEST8REALM"
+        cmd = self.master.run_command(
+            [
+                "ipa-adtrust-install",
+                "-a",
+                self.master.config.admin_password,
+                "--netbios-name",
+                netbios_name,
+                "-U",
+            ]
+        )
+        assert cmd.returncode == 0
```

Comment:
You can remove this assert (run_command was called without raiseonerr, and it defaults to True, hence any error would raise an exception).